Makefile

```diff
@@ -30,3 +30,6 @@ set-token:
 
 publish:
 	go run -tags=staging -ldflags="-X github.com/datreeio/datree/cmd.CliVersion=0.0.1" main.go publish ./internal/fixtures/policyAsCode/valid-schema.yaml
+
+datree_test_to_JUnit_report: # install junit2html first: https://github.com/inorton/junit2html
```

Comment:
this is nice but each developer who will want to use it will have to download the pkg first which can be confusing w/o looking into this file